corehq/apps/cleanup/tasks.py

```diff
@@ -40,9 +40,15 @@ def permanently_delete_eligible_data(dry_run=False):
 
     logger.info(f"{dry_run_tag}'permanently_delete_eligible_data' ran with the following results:\n")
     for table, count in form_counts.items():
-        logger.info(f"{dry_run_tag}{count} {table} objects were deleted.")
+        if table == 'tombstone':
+            logger.info(f"{count} XFormInstance {table}s were created.")
```

Comment:
Why is `{dry_run_tag}` not used here?

Reply:
tombstones don't need to and won't be created for dry runs - that's just to get a sense of how many forms/cases will be deleted during the next run